upgrade-cluster.yml

```diff
@@ -78,10 +78,19 @@
     - { role: kubernetes/master, tags: master }
     - { role: kubernetes/client, tags: client }
     - { role: kubernetes-apps/cluster_roles, tags: cluster-roles }
-    - { role: network_plugin, tags: network }
     - { role: upgrade/post-upgrade, tags: post-upgrade }
   environment: "{{proxy_env}}"
 
+#Upgrade calico on all masters and nodes
+- hosts: kube-master:kube-node
```

Comment:
This will break the behavior where you can upgrade and/or install a new node in upgrade playbook. I'm not entirely opposed, but it's worth updating the docs to indicate this.

Reply:
i had problems when if you do like previous way, network plugin is run on masters, then on nodes, it breaks all the upgrade procedure, i tried to think of better way but was unsuccessful.